src/HanziWriter.js

```diff
@@ -178,6 +179,10 @@ HanziWriter.prototype.setCharacter = function(char) {
 
     if (this._options.showCharacter) this._characterRenderer.showImmediate();
     if (this._options.showOutline) this._outlineRenderer.showImmediate();
+  }, () => {
+    if (this._options.onCharDataLoaderError) {
```

Comment:
There's a helper in `./utils` called `callIfExists()` that you can use here. It would look something like:
`callIfExists(this._options.onCharDataLoaderError, ...args)`

Reply:
nice !